tests/test_devices.py

```diff
@@ -999,3 +1004,99 @@ async def test_device_websocket(
 def test_enum_unknowns() -> None:
     """Validate enum unknown values."""
     assert DeviceState(999) == DeviceState.UNKNOWN
+
+
+@pytest.mark.parametrize(
+    ("device_payload", "data", "command"),
+    [
```

Comment:
Also add test coverage for the cases where you only use either brightness or color